amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -15,6 +15,11 @@
       <app.amp.client.war.folder>${project.build.directory}/${project.build.finalName}-war</app.amp.client.war.folder>
 
       <image.name>alfresco/alfresco-governance-repository-community-base</image.name>
+
+      <local.registry>127.0.0.1:5000</local.registry>
+      <base.image>alfresco/alfresco-community-repo-base</base.image>
+      <base.image.tag>${image.tag}</base.image.tag>
```

Comment:
Is there a reason why we have this `base.image.tag` that just points to `image.tag` anyways? Which process will require setting this to a different value than `image.tag`?

Reply:
No, I deleted it.